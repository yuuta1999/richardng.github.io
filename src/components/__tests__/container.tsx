import React from 'react'
import 'jest-styled-components'
import { render } from '@testing-library/react'

import Container from '../container'

describe('Container', () => {
  const { container } = render(<Container />)

  it('it should works', () => {
    expect(container.firstChild).toMatchSnapshot()
  })
})
