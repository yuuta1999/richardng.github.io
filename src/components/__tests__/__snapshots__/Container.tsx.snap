// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container it should works 1`] = `
.c0 {
  width: 100%;
  margin-right: auto;
  margin-left: auto;
  padding-right: 1rem;
  padding-left: 1rem;
}

@media (min-width:576px) {
  .c0 {
    max-width: 540px;
  }
}

@media (min-width:768px) {
  .c0 {
    max-width: 720px;
  }
}

@media (min-width:992px) {
  .c0 {
    max-width: 960px;
  }
}

@media (min-width:1200px) {
  .c0 {
    max-width: 1040px;
  }
}

<div
  class="c0"
/>
`;
